vllm_spyre/worker/spyre_worker.py

```diff
@@ -35,6 +36,20 @@
 # yapf: enable
 
 
+# Capture a oneshot metric (e.x warmup)
+def _spyre_metric_oneshot(name, labels, value):
+    name = "vllm:plugin:spyre:" + name
+    # Cleanup existing, for CI.
```

Comment:
Why is this cleanup required?

Reply:
The prometheus client is independent of the vllm objects being created and destroyed while inside the same python vm. The original PR didn't clean up, and the result was many failures due to already registered gauges. vllm handles this in a similar way.